KinConnections/resources/email_wrapper.py

```diff
@@ -0,0 +1,96 @@
+# email_wrapper.py - KinConnections
+# Serves as wrapper for sending connection emails
+
+# for environment variables
+import os
+from dotenv import load_dotenv
+# for email engine
+import smtplib, ssl
+from email.mime.text import MIMEText
+from email.mime.multipart import MIMEMultipart
+from email.utils import formataddr
+# for rich html email
+from dominate import document
+from dominate.tags import *
+# flask
+from flask import Blueprint, Response, request, render_template, session, redirect, url_for
+from flask_restful import Resource
+
+from .db_wrapper import *
+
+
+# access credentials from .env
+load_dotenv(dotenv_path="../.env")
+
+kc_email = os.getenv("KC_EMAIL")
+kc_email_password = os.getenv("KC_EMAIL_PASSWORD")
+kc_email_server = os.getenv("KC_EMAIL_SERVER")
+kc_email_port = int(os.getenv("KC_EMAIL_PORT"))
+
+class Email(Resource):
+    def get(self):
+        return 'Error! Return <a href="/">Home</a>'
+        
+    def post(self, id):
+        # ensure logged in
+        if not session.get('email', None):
+            return redirect("/login?error=notSignedIn")
+
+        # send email
+        subject = (request.form.get('userSubject'))
+        message = (request.form.get('userMessage'))
+
+        currentConnector = db_wrapper.get_connector_by_id(id)
+        connector_name = (currentConnector['first_name'] + ' ' + currentConnector['last_name'])
+        connector_email = (currentConnector['email'])
+
+        sender_name = (session['first_name'] + ' ' + session['last_name'])
+        sender_email = (session['email'])
+
+        if (self.__send_email_message(sender_name, sender_email, connector_name, connector_email, subject, message)):
+            return "OK"
+        else:
+            return "Message not sent!"
+
+    def __send_email_message(self, sender_name, sender_email, recipient_name, recipient_email, message_subject, message_body):
+        
+        message = MIMEMultipart("alternative")
+
+        message['Subject'] = "[KinConnections] New Connection from " + sender_name
+        message['From'] = formataddr(("Kin Connections", "connections@kc.campconnect.co"))
```

Comment:
I wonder if it's worth it to make `Kin Connections` and `connections@kc.campconnect.co` variables inside `.env`. I am thinking about cases where we might need to change these but I doubt that'll be often. This might just be my brain freaking out over seeing hardcoded values. I'll let you make the decision on this.